lib/rules/decorator-position.js

```diff
@@ -0,0 +1,157 @@
+'use strict';
+
+const ember = require('./ember');
+const utils = require('./utils');
+
+const defaultOptions = {
+  trackedOnSameLine: true,
```

Comment:
I may want to make this more generic, and have something like:

```
{
  onSameLine: [`@tracked`, `@action`, [`@computed`, { withArgs: false }], `@attr`, `@hasMany`, `@belongsTo`],
  onDifferentLines: [`@dependentKeyCompat`,  [`@computed`, { withArgs: true }]],
}
```

that way, this could be augmented for people's custom decorators, as well.